### ISSUE TITLE
Get a more powerful ARM 32 box

### ISSUE THREAD

**Issue by karianna** [AdoptOpenJDK/openjdk-build#78]

**Comment by gdams** [comment]
@vielmetti is working on sorting out ARM32 on Packet

**Comment by vielmetti** [comment]
In particular, the ARM32 boxes we have report that they are ARM64 machines but they also run ARM32 binaries, so getting your build environment just exactly right might require some finesse. I am talking to folks who have been through this so that I can give you some guidance, look for something next week if all goes well.
```
# uname -p
aarch64
```

**Comment by tellison** [comment]
Any progress here?  I see we are configuring up a new ARM32 machine at the moment, but I believe it is not any "more powerful" than the earlier machine.

**Comment by gdams** [comment]
Hopefully @vielmetti is going to be able to help us out with more powerful arm32 hardware but it is currently not ready for release in packet from what I understand

**Comment by gouessej** [comment]
Sorry for the silly question but would it help to build OpenJDK for 32-bit machines? I see no OpenJDK binary build for 32-bit Intel and 32-bit ARM GNU Linux, Apple OS X and Microsoft Windows.

**Comment by karianna** [comment]
@gouessej I think the only viable 32-bit target is ARM these days.  I doubt anyone is running a 32-bit intel...

**Comment by gouessej** [comment]
@karianna Of course, as time goes by, less and less people are running 32-bit Intel but numerous libraries still provide 32-bit binaries and Oracle still provide 32-bit JREs:
http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html

I'd like to use AdoptOpenJDK instead of some old binary builds made by alexkasko whose project is discontinued. I see that ojdkbuild provides 32-bit builds too:
https://github.com/ojdkbuild/ojdkbuild

Is it really a big deal?

**Comment by karianna** [comment]
It depends on whether we can get 32 bit hosts.  We rely on the generosity of IaaS providers :-)
